api/src/game/utils.ts

```diff
@@ -0,0 +1,144 @@
+import { IBoard, IGameStates, TPlayer } from './types';
+
+export const getInitialBoardState = (player: TPlayer): IBoard => {
+  const player1Board = {
+    A1: { name: 'spy', value: 99 },
+    A2: { name: 'spy', value: 99 },
+    A3: { name: 'flag', value: 1 },
+    A4: { name: 'private', value: 2 },
+    A5: { name: 'private', value: 2 },
+    A6: { name: 'private', value: 2 },
+    A7: { name: 'private', value: 2 },
+    A8: { name: 'private', value: 2 },
+    B1: { name: 'private', value: 2 },
+    B2: { name: 'sergeant', value: 3 },
+    B3: { name: '2ndlt', value: 4 },
+    B4: { name: '1stlt', value: 5 },
+    B5: { name: 'captain', value: 6 },
+    B6: { name: 'major', value: 7 },
+    B7: { name: 'ltcol', value: 8 },
+    B8: { name: 'col', value: 9 },
+    C1: { name: '1star', value: 10 },
+    C2: { name: '2star', value: 11 },
+    C3: { name: '3star', value: 12 },
+    C4: { name: '4star', value: 13 },
+    C5: { name: '5star', value: 14 }
+  };
+
+  const player2Board = {
+    I1: { name: 'spy', value: 99 },
+    I2: { name: 'spy', value: 99 },
+    I3: { name: 'flag', value: 1 },
+    I4: { name: 'private', value: 2 },
+    I5: { name: 'private', value: 2 },
+    I6: { name: 'private', value: 2 },
+    I7: { name: 'private', value: 2 },
+    I8: { name: 'private', value: 2 },
+    H1: { name: 'private', value: 2 },
+    H2: { name: 'sergeant', value: 3 },
+    H3: { name: '2ndlt', value: 4 },
+    H4: { name: '1stlt', value: 5 },
+    H5: { name: 'captain', value: 6 },
+    H6: { name: 'major', value: 7 },
+    H7: { name: 'ltcol', value: 8 },
+    H8: { name: 'col', value: 9 },
+    G1: { name: '1star', value: 10 },
+    G2: { name: '2star', value: 11 },
+    G3: { name: '3star', value: 12 },
+    G4: { name: '4star', value: 13 },
+    G5: { name: '5star', value: 14 }
+  };
+
+  return (player === 'p1') ? player1Board : player2Board;
+};
+
+/**
+ * Receives the players' boards and `cleans` them, that is, removing
+ * the values of the opponent's pieces in order for the data to be
+ * published or sent to the player (because otherwise, the player
+ * would then know the values of the opponent's pieces which would
+ * make the game pointless)
+ *
+ * @param {IBoard} b1 First board (from player 1)
+ * @param {IBoard} b2 Second board (from player 2)
+ * @returns {Object} An object with `b1` and `b2` (no values) combined, an object with `b1` (no values) and `b2` combined, and an object that contains `b1` and `b2` with both having no values
```

Comment:
You wrote the same thing twice.

```suggestion
 * @returns {Object} An object with `b1` and `b2` (no values) combined. 
```

Reply:
It returns three boards. One is b1 with b2 (no values), another is b1 (no values) with b2, last is b1 + b2 (both no values)